web/src/components/profile/ProfileAddressLockedFields.test.tsx

```diff
@@ -0,0 +1,75 @@
+import { ProfileAddressLockedFields } from "@/components/profile/ProfileAddressLockedFields";
+import { generateAddress } from "@/test/factories";
+import { WithStatefulAddressData } from "@/test/mock/withStatefulAddressData";
+import { Address, emptyAddressData, Municipality } from "@businessnjgovnavigator/shared/";
+import { generateMunicipality } from "@businessnjgovnavigator/shared/test";
+import { render, screen, within } from "@testing-library/react";
+
+describe("ProfileAddressLockedFields", () => {
+  const renderComponent = ({
+    address,
+    municipalities,
+  }: {
+    address?: Address;
+    municipalities?: Municipality[];
+  }): void => {
+    render(
+      <WithStatefulAddressData
+        initialData={address || emptyAddressData}
+        municipalities={municipalities || [generateMunicipality({})]}
+      >
+        <ProfileAddressLockedFields />
+      </WithStatefulAddressData>
+    );
+  };
+
+  it("renders profile address locked fields without Address Line 2", () => {
+    const address = generateAddress({
+      addressLine1: "1111 Home Alone",
+      addressLine2: "",
+      addressMunicipality: generateMunicipality({ displayName: "Allendale" }),
+      addressState: { shortCode: "NJ", name: "New Jersey" },
+      addressZipCode: "00893",
+    });
+    renderComponent({ address });
+
+    const addressLine1 = screen.getByTestId("locked-profileAddressLine1");
+    expect(within(addressLine1).getByText("1111 Home Alone")).toBeInTheDocument();
+    const addressMuniStateZip = screen.getByTestId("locked-profileAddressMuniStateZip");
+    expect(within(addressMuniStateZip).getByText("Allendale, NJ 00893")).toBeInTheDocument();
+    expect(screen.queryByTestId("locked-profileAddressLine2")).not.toBeInTheDocument();
+  });
+
+  it("renders profile address locked fields with Address Line 2", () => {
```

Comment:
pebble, I think only one test is required around address line 2 being displayed when it's available

Reply:
There is only one test around address line 2 being displayed when it's available. We've updated the title of the first test to minimize confusion.